kratos/includes/dof.h

```diff
@@ -477,7 +477,11 @@ class Dof{
     int mIndex : 6;
 
     /** Equation identificator of the degree of freedom */
+#if defined (__arm__)
```

Comment:
```suggestion
#if defined (__arm__) // required to avoid overflow on ARM systems (32 bit)
```

Reply:
OK, but I will add a capital at the beginning

Reply:
I would go for defining a macro for 32bit compilation and setting it in defines.h rather than using the `__arm__` macro here.

Reply:
I though we drop support for x86. If we want to define macro fro 32 bits it will require many cases, as this depends on the compiler and the OS

Reply:
afaik for linux we still support it (?)

Reply:
I updated, now it is general for 32 bits (maybe be not complete, the list cannot be completed and depends on compiler and OS). @philbucher can you check it works on Intel compiler? 

See: https://sourceforge.net/p/predef/wiki/Architectures/